### PR TITLE
Update AndroidPolice.com

### LINKED ISSUE #14602: Androidpolice.com ruleset
Type: ruleset issue

Domain: androidpolice.com
Domain: www.androidpolice.com

Androidpolice is now HTTPS by default.

### PR DESCRIPTION
**Pull request by DavidLiedke**
fix #14602

### CODE REVIEW

**Comment by Bisaloo** [comment]
@DavidLiedke, thank you for taking care of #14602! Did you know you can auto-close issues when you propose a fix but appending fix/fixes/close/closes/... to the issue number?

https://help.github.com/articles/closing-issues-using-keywords/

It would be really helpful if you could edit your first message to use this feature and use it in the future as well.

**Comment by DavidLiedke** [comment]
@Bisaloo Thanks for this information.

I edit the pull comment.

File: src/chrome/content/rules/AndroidPolice.xml [hunk 1]
```diff
@@ -1,16 +1,6 @@
...
-
-Disabled by https-everywhere-checker because:
-Fetch error: http://www.androidpolice.com/ => https://www.androidpolice.com/: Cycle detected - URL already encountered: https://www.androidpolice.com/
-Fetch error: http://androidpolice.com/ => https://www.androidpolice.com/: Cycle detected - URL already encountered: https://www.androidpolice.com/
--->
-<ruleset name="Androidpolice" default_off='failed ruleset test'>
+<ruleset name="AndroidPolice.com">
   <target host="www.androidpolice.com"/>
   <target host="androidpolice.com"/>
   <rule from="^http://(?:www\.)?androidpolice\.com/" to="https://www.androidpolice.com/"/>
```
Comment (J0WI) [inline src/chrome/content/rules/AndroidPolice.xml]: This can be simplified.

**Comment by DavidLiedke** [comment]
@J0WI done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**